packages/webworker/lumberjack/src/lib/logging/create-lumberjack-log-factory.ts

```diff
@@ -7,10 +7,15 @@ interface LumberjackLogFactoryDependencies {
   getUnixEpochTicks: () => number;
 }
 
-export type LumberjackLogFactory<TPayload extends LumberjackLogPayload | void = void> = ReturnType<
-  typeof createLumberjackLogFactory<TPayload>
->;
-
+/**
+ * Factory function to create a set of functions for building Lumberjack logs with different levels.
```

Comment:
nit: Use imperative tense.
```suggestion
 * Create a set of functions for building Lumberjack logs of distinct levels.
```